src/ApparentHorizons/Tags.hpp

```diff
@@ -344,11 +344,12 @@ struct UnitNormalVectorCompute : UnitNormalVector<Frame>, db::ComputeTag {
   using return_type = tnsr::I<DataVector, 3, Frame>;
 };
 
-///SpatialRicciScalar is the two-dimensional intrinsic Ricci scalar curvature
+/// Ricci scalar is the two-dimensional intrinsic Ricci scalar curvature
 /// of a Strahlkorper
```

Comment:
Sorry, I meant please rename the tags from `SpatialRicciScalar` to `RicciScalar`

Reply:
Updated.